HPM/localisation/00_HPM_countries.csv

```diff
@@ -806,13 +820,20 @@ BAH_fascist;Partido Fascist;Partido fasciste;Partido Fascist;;;;;;;;;;;
 BAN;Banat;;;;;;;;;;;;;
 BAN_democracy;Banat Republic;;;;;;;;;;;;;
 BAN_ADJ;Banati;;;;;;;;;;;;;
-BAN_conservative;The Conservatives;Faction royale;K�nigsfraktion;;;;;;;;;;;
-BAN_liberal;The Liberals;Faction lib�rale;Die liberale Fraktion;;;;;;;;;;;
+BAN_conservative;Deutsche Partei;;;;;;;;;;;;;
+BAN_liberal;Liberal Party;Faction lib�rale;Die liberale Fraktion;;;;;;;;;;;
 BAN_reactionary;The Royalist Faction;Faction nationaliste;Die nationalistische Fraktion;;;;;;;;;;;
 BAN_anarcho_liberal;The Libertains;Faction radicale;Die radikale Fraktion;;;;;;;;;;;
-BAN_socialist;The Socialists;Faction socialiste;Die sozialistische Fraktion;;;;;;;;;;;
-BAN_communist;The Communists;Faction communiste;Die kommunistische Fraktion;;;;;;;;;;;
-BAN_fascist;The Nationalist Faction;Faction fasciste;Die faschistische Fraktion;;;;;;;;;;;
+BAN_socialist;Banat Sozialdemokratische Partei;;;;;;;;;;;;;
+BAN_communist;Communist Party;Faction communiste;Die kommunistische Fraktion;;;;;;;;;;;
+BAN_fascist;Nationalsozialistische Partei der rum�nischen Deutschen;;;;;;;;;;;;;
```

Comment:
Are „die rumänischen Deutsche” and variations attested? For the most part, I’m finding hits when looking for [„die Rumäniendeutsche”](https://www.google.com/search?q=Rum%C3%A4niendeutsche&tbm=bks). Because those hits are mostly from post-WWII documents, I have no idea if the term was contemporaneous to the time period covered by the game or not.

Reply:
Since the name is too long, I'll change it to "Deutsche Volkspartei" which was another fascist party from the same period.